ui/appui-react/package.json

```diff
@@ -138,6 +138,7 @@
     "@itwin/itwinui-css": "0.x",
     "@itwin/itwinui-react": "~1.42.0",
     "@itwin/itwinui-icons-react": "^1.8.0",
+    "@itwin/itwinui-icons": "^1.14.0",
```

Comment:
Yes, because we need it to be a string for now. I hope to remove that in the webfont icon cleanup by creating a ConditionalIcon to match ConditionalStringValue.
I think the ^ ensure they'll  both have the same major release and the latest minor. I think there are more versions of icons than of icons-react (there wasn't a 1-1 release of react for the other package).

Reply:
the itwinui-icons, and itwinui-icons-react pkgs are published in lockstep. 
the ^ here ensures theyre the same version, but yea its best to keep them in sync imo